docs/getting_started.md

```diff
@@ -0,0 +1,661 @@
+└───────────┴───────────┴──────────────────────┴────────────────────────┘
+```
+
+If we need more specific groups, we can add to the `group_by`.
+
+```python
+>>> penguins.group_by(["species", "island", "sex"]).aggregate(
+...:     [penguins.bill_length_mm.mean(), penguins.flipper_length_mm.max()]
+...: )
+┏━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ species ┃ island    ┃ sex    ┃ Mean(bill_length_mm) ┃ Max(flipper_length_mm) ┃
+┡━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━┩
+│ string  │ string    │ string │ float64              │ int64                  │
+├─────────┼───────────┼────────┼──────────────────────┼────────────────────────┤
+│ Adelie  │ Torgersen │ male   │            40.586957 │                    210 │
+│ Adelie  │ Torgersen │ female │            37.554167 │                    196 │
+│ Adelie  │ Torgersen │ ∅      │            37.925000 │                    193 │
+│ Adelie  │ Biscoe    │ female │            37.359091 │                    199 │
+│ Adelie  │ Biscoe    │ male   │            40.590909 │                    203 │
+│ Adelie  │ Dream     │ female │            36.911111 │                    202 │
+│ Adelie  │ Dream     │ male   │            40.071429 │                    208 │
+│ Adelie  │ Dream     │ ∅      │            37.500000 │                    179 │
+│ Gentoo  │ Biscoe    │ female │            45.563793 │                    222 │
+│ Gentoo  │ Biscoe    │ male   │            49.473770 │                    231 │
+│ …       │ …         │ …      │                    … │                      … │
+└─────────┴───────────┴────────┴──────────────────────┴────────────────────────┘
+```
+
+## Chaining it all together
+
+We've already chained some Ibis calls together. We used `mutate` to create a new
+column and then `select` to only view a subset of the new table. We were just
+chaining `group_by` with `aggregate`.
+
+There's nothing stopping us from putting all of these concepts together to ask
+questions of the data.
+
+How about:
+
+- What was the largest female penguin (by body mass) on each island in the year 2008?
+
+```python
+>>> penguins.filter([penguins.sex == "female", penguins.year == 2008]).group_by(
+...:     ["island"]
+...: ).aggregate(penguins.body_mass_g.max())
+┏━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ island    ┃ Max(body_mass_g) ┃
+┡━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ string    │ int64            │
+├───────────┼──────────────────┤
+│ Biscoe    │             5200 │
+│ Torgersen │             3800 │
+│ Dream     │             3900 │
+└───────────┴──────────────────┘
+```
+
+- What about the largest male penguin (by body mass) on each island for each year of data collection?
+
+```python
+>>> penguins.filter([penguins.sex == "male"]).group_by(["island", "year"]).aggregate(
+...:     penguins.body_mass_g.max().name("max_body_mass")
+...: ).order_by(["year", "max_body_mass"])
```

Comment:
Many of these appear off by a single space. Can you remove the `:` after the `...`?

Reply:
Yep -- also removed all the `...:` -- I wanted these to be copy-pasteable so I overrode the `ipython` prompt, but forgot about the continuation lines